ethcore/src/miner/miner.rs

```diff
@@ -292,8 +292,8 @@ impl Miner {
 		for tx in transactions {
 			let hash = tx.hash();
 			match open_block.push_transaction(tx, None) {
-				Err(Error::Execution(ExecutionError::BlockGasLimitReached { gas_limit, gas_used, .. })) => {
-					debug!(target: "miner", "Skipping adding transaction to block because of gas limit: {:?}", hash);
+				Err(Error::Execution(ExecutionError::BlockGasLimitReached { gas_limit, gas_used, gas })) => {
```

Comment:
not sure if it is correct to return Error _and_ logging tit?
for me returned error is enough here while it's invoking code that should log